modules/ml/src/main/java/org/apache/ignite/ml/environment/logging/ConsoleLogger.java

```diff
@@ -82,6 +82,9 @@ private void print(VerboseLevel verboseLevel, String line) {
      * ConsoleLogger factory.
      */
     private static class Factory implements MLLogger.Factory {
+        /** Serial version uuid. */
+        private static final long serialVersionUID = 5864605548782107893L;
```

Comment:
It seems that you forget to extend Serializable i-face in MLLogger.Factory

Reply:
No, but `LearningEnvironmentBuilder::withLoggingFactory` signature is changed.